app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -79,18 +80,24 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
   private val bloodPressureSection = Section()
   private val medicalHistorySection = Section()
   private val adapterUiEvents = PublishSubject.create<UiEvent>()
+  private val bpWasSavedEvents = PublishSubject.create<UiEvent>()
 
   private var bpEntrySheetAlreadyShownOnStart: Boolean = false
+  private var bloodPressureWasSaved: Boolean = false
 
   override fun onSaveInstanceState(): Parcelable {
     return PatientSummaryScreenSavedState(
         super.onSaveInstanceState(),
-        bpEntryShownOnStart = bpEntrySheetAlreadyShownOnStart)
+        bpEntryShownOnStart = bpEntrySheetAlreadyShownOnStart,
+        bloodPressureWasSaved = bloodPressureWasSaved)
```

Comment:
Can we use the same tone for both `bpEntryShownOnStart` and `bloodPressureWasSaved`?

Reply:
If you decide to change `bloodPressureWasSaved` then `bpWasSavedEvents` will also need to be updated